backEnd/prisma/seed.ts

```diff
@@ -0,0 +1,54 @@
+import prisma from "../src/prismaClient";
+import { defaultUsers, menuCategories, menuItems } from './seedData'
+
+async function main() {
+    // users
+    await Promise.all(defaultUsers.map( async (user) => {
+        return prisma.user.create({
+            data: {
+                username: user.username,
+                firstName: user.firstName,
+                lastName: user.lastName,
+                password: user.password,
```

Comment:
Right now, this stores the plain text password in the database. We will want to use the same bcrypt hash mechanism that we use in our signup route so we can login with the passwords set here.

Reply:
Whoops, can't believe I forgot that.